README.md

```diff
@@ -1,9 +1,11 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2021S1-CS2103T-T11-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
need to change https://github.com/se-edu/addressbook-level3/actions to this https://github.com/AY2021S1-CS2103T-T11-2/tp/actions

Reply:
Hmm the old link generates a badge though. Using `/actions/` doesn't generate a badge, but a link instead...

Reply:
Ohh I see sorry I misunderstood that.